index.js

```diff
@@ -184,25 +185,49 @@ function resolveWebpackPath(partial, filename, directory, webpackConfig) {
 
   try {
     var loadedConfig = require(webpackConfig);
-    var resolver = webpackResolve.create.sync(loadedConfig.resolve || {});
+  } catch (e) {
+    debug('error loading the webpack config at ' + webpackConfig);
+    debug(e.message);
+    debug(e.stack);
+    return '';
+  }
+
+  try {
```

Comment:
Non-blocker: we technically don't need all of this code in the `try` block. Only two statements might throw:

`webpackResolve.create.sync(resolveConfig);` and `resolver(lookupPath, partial)`